internal/runnerinstall/ecs.go

```diff
@@ -160,12 +158,26 @@ func (i *ECSRunnerInstaller) Install(ctx context.Context, opts *InstallOpts) err
 				return err
 			}
 
-			if netInfo, err = awsinstallutil.SetupNetworking(ctx, ui, sess, i.Config.Subnets); err != nil {
+			// TODO: Add a port here if the user specified the -liveness-tcp-addr flag
+			// after `--` on the runner install CLI - currently without this, there
+			// is no way to use that flag effectively, since the SG won't allow traffic
+			// to the port of the -liveness-tcp-addr flag
+			if netInfo, err = awsinstallutil.SetupNetworking(ctx, ui, sess, i.Config.Subnets, []*int64{aws.Int64(int64(2049))}); err != nil {
 				return err
 			}
 			i.netInfo = netInfo
 
-			if efsInfo, err = awsinstallutil.SetupEFS(ctx, ui, sess, netInfo); err != nil {
+			efsTags := []*efs.Tag{
+				{
+					Key:   aws.String(DefaultRunnerTagName),
+					Value: aws.String(defaultRunnerTagValue),
+				},
+				{
+					Key:   aws.String("runner-id"),
```

Comment:
> why is DefaultRunnerTagName exported but defaultRunnerTagValue is not?
Perhaps at one point it may have been used in the `installutil` or `serverinstall` packages, but it appears that if that was the case, it no longer is. Will change to unxported.

Also, I've updated "runner-id" to be a const called "defaultRunnerIdTagName".